roles/matrix-bridge-hookshot/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+- import_tasks: "{{ role_path }}/tasks/init.yml"
+  tags:
+    - always
+
```

Comment:
Have you thought about using some `validate_config.yml` file here to do configuration validation?

You could have tasks that check per-service configuration. For example, if Figma is enabled (`matrix_hookshot_figma_enabled`), ensure that `matrix_hookshot_figma_instances is defined` and `matrix_hookshot_figma_instances|length > 0`, otherwise complain.

-----

Maybe we don't need to do this though, because you're already invoking the Hookshot bridge itself to validate the configuration. And the bridge can do a much more thorough job at it anyway.

Still, if you think that we can catch common mistakes and provide better error messages (compared to the output spewed by the bridge), it may make sense to add a validation task of our own.

Reply:
I decided that checking if relevant values for enabled services are filled in is easy enough to do, especially I can barely get the upstream validation to fail even with obvious misconfiguration.